src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -0,0 +1,495 @@
+package com.ncc.aif;
+
+import javafx.util.Pair;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFFormat;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.ncc.aif.AIFUtils.*;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Utilities for testing AIF functionality and/or creating examples.
+ * Features:
+ * <p><ul>
+ * <li>Contains methods for creating a valid AIF entity, event, relation, and hypothesis objects.;</li>
+ * <li>Objects, assertions, and cluster URIs are numbered in a standard, sequential, simple way;</li>
+ * <li>Is entirely ontology-agnostic;</li>
+ * <li>Contains methods for asserting test model validity and dumping models.</li>
+ * </ul></p>
+ * Call {@link #startNewTest()} before each test to ensure a clean model.
+ */
+class TestUtils {
```

Comment:
I'm OK with having this be one large file, but I know some people have trouble reading such files. Was there a reason you didn't want to add each class as its own file?

Reply:
You know, if we treat `ExamplesAndValidationTest.java` as a cautionary tale, then yes, we should absolutely break this up into three files!  Having it as one file is sometimes easier to deal with, but there are valid/solid reasons to break it up (again, see `ExamplesAndValidationTest.java`).

Like you, I think I'm fine either way.  Do @craigwarsaw and @sharknc have an opinion on this?

Reply:
I have split the classes into three separate files.